README.md

```diff
@@ -452,6 +470,38 @@ end
 
 ---
 
+# ![](http://placehold.it/890x200/2b1e34/ffffff?text=-%20%20%20%20%20%20%20Notifications%20%20%20%20%20%20%20-)
+
+Notifications/alerts allow us to talk with the user about the various aspects of the editting experience. For us, the contributors, this means that we may warn the user or developer of a breaking change that requires migrations (if they were forgotten to be executed). Since this is configurable, the developer may elect to push custom notifications to various parts of the Thesis editor.
+
+![](https://user-images.githubusercontent.com/1775841/38072424-07657800-32db-11e8-8fe1-bdea4bfc76ab.png)
+
+Notifications can be configured to be static:
+
+```elixir
+config :thesis, :notifications,
+  page_settings: ["Example notification 1", "Example notification 2"],
+  add_page: ["Example notification 3"],
+  import_export_restore: ["Example notification 4"]
+```
+
+Or, you may elect to add some logic and make them more dynamic:
+
+```elixir
+config :thesis, :notifications,
+  page_settings: ["Example notification 1", "Example notification 2"],
+  add_page: &MyApp.CustomModule.generate_notifications/0,
+  import_export_restore: &MyApp.CustomModule.import_warning/0
+```
+
+In either case, the only thing that matters is that you provide a List of String(s), whether static or the result of a custom function ([example](https://github.com/infinitered/thesis-phoenix/blob/master/examples/example-phx-1_3/lib/example_phx/notifications.ex)).
+
+<strong>Note</strong>: right now, there are 3 spots to which you can push notifications: the '[Add New Page](https://user-images.githubusercontent.com/1775841/38072348-acfb3d46-32da-11e8-8fed-97b5d17ba027.png)' tray, '[Page Settings](https://user-images.githubusercontent.com/1775841/38072314-8b056a86-32da-11e8-8e75-df4383eabdee.png)' tray, and '[Import/Export/Restore](https://user-images.githubusercontent.com/1775841/38072274-67b43a62-32da-11e8-8752-664930d4ce9d.png)' tray. As more features are developed, the notifications will be extended to support those features as well.
+
+<br/>
+
+---
+
```

Comment:
Adds a new section for documenting Notifications:
![20180328_zey2r](https://user-images.githubusercontent.com/1775841/38073408-b83a5c82-32df-11e8-8b2a-e36eaca70c17.png)